docs/source/nutshell.rst

```diff
@@ -0,0 +1,223 @@
+=========================
+|langname| in a nutshell
+=========================
+
+.. highlight:: btm
+
+|langname| features a simple syntax to express Bitcoin transactions. 
+We illustrate  it through a series of examples, that you can experiment with in the `web editor <http://blockchain.unica.it/btm/>`_.
+
+
+.. _label_t_modeling:
+
+"""""""""""""""""""""
+A basic transaction 
+"""""""""""""""""""""
+
+Bitcoin transactions transfer currency, the *bitcoins* (BTC).
+Each transaction has one or more inputs, from where it takes the bitcoins,  and one or more outputs, which specify the recipient(s).
+|langname| also allows for  transactions  with  no inputs:
+even thought these transactions cannot be appended *as is* to the actual
+Bitcoin blockchain, they are useful to  refer to transactions which are
+not know at specification time. 
+An example of transaction with no inputs is the following:  
+
+
+.. code-block:: btm
+
+	transaction T {
+		input = _    // no input 
+		output = 50 BTC: fun(x) . x==42
+	}
+
+The output field of transaction ``T`` contains a value, ``50 BTC``, and 
+an *output script*,  ``fun(x) . x==42``.
+This means that  50 bitcoins will  be transferred to any transaction
+which provides a *witness*  ``x``  such that ``x == 42``.
+
+To append ``T`` to the Bitcoin blockchain,
+the placeholder ``_`` for the input must be replaced with the identifier
+of an unspent transaction already on the blockchain,
+which has at least 50 BTC.  
+
+You can use the `web editor <http://blockchain.unica.it/btm/>`_  to write
+|langname| transactions,   to check their syntax, and to compile them  into
+actual Bitcoin  transactions.
+The output of the compiler is a serialized transaction for the Bitcoin
+testnet.
+To generate transactions for the main net, one can specifying it as follows:  
+
+	.. code-block:: btm
+
+	
+		network mainnet  // default testnet
+
+
+For instance, let us paste transaction ``T`` into the editor and then let us add command ``compile T`` to it. 
+Now, if we hit the button [Compile], the web editor shows in the output box the transaction ``T``  in  Bitcoin (testnet) serialization format.
+
+.. figure:: _static/img/compiling_t.png
+	:scale: 100 %
+	:class: img-border
+	:align: center
+
+The serialized transaction can  be send to the Bitcoin network using the Bitcoin client command ``bitcoin-cli sendrawtransaction``.
+Please note, that before sending it, the unspecified input must be substituted by an actual transaction identifier, like , for instance:
```

Comment:
transaction identifier, like , for instance: -> transaction identifier, like:
or
transaction identifier, like , for instance: -> transaction identifier, for instance:

Reply:
ok